README.rst

```diff
@@ -50,7 +50,8 @@ Exporting and importing sets of repositories
 Vcstool can export and import all the information required to reproduce the versions of a set of repositories.
 Vcstool uses a simple `YAML <http://www.yaml.org/>`_ format to encode this information.
 This format includes a root key ``repositories`` under which each local repository is described by a dictionary keyed by its relative path.
-Each of these dictionaries contains keys ``type``, ``url``, and ``version``.
+Each of these dictionaries contains keys ``type``, ``url``, and ``version``. The optional ``subpaths`` key may be used
```

Comment:
No-blocking, just saying; Since underlining functionality (in `git`) is `sparse-checkout`, naming it differently may not be intuitive. However, `subpath` makes more sense in the context of attribute name `.repos` file (and in this context, `sparse-checkout` wouldn't sound reasonable).

Reply:
The documentation refers to them as "patterns", which I use elsewhere in the code.

> Write a set of patterns to the sparse-checkout file, as given as a list of arguments following the set subcommand. Update the working directory to match the new patterns.

I could be more explicit and call them `sparse-checkout-patterns`. This could lead to more complexity since in the git call I used `--cone` to restrict patterns to directories only (see https://git-scm.com/docs/git-sparse-checkout#_cone_pattern_set), and I'd either need a way to indicate that non-cone patterns aren't supported or I'd need to add support for non-cone clones.

Reply:
Another note, though, is that this type of action could be supported in other vcs. Calling it after the git specific command could cause problems in the future.